internal/ctrlutils/ingress-status.go

```diff
@@ -325,11 +337,15 @@ func UpdateUDPIngress(ctx context.Context, logger logr.Logger, svc file.FService
 				break
 			}
 			if errors.IsNotFound(err) {
+				log.Debugf("failed to update UDPIngress status because the object (%s/%s) is gone, status update stopped.", namespace, name)
 				return nil
 			}
-
-			log.Errorf("failed to update UDPIngress status: %v. retry...", err)
-			time.Sleep(time.Second)
+			if errors.IsConflict(err) {
+				log.Debugf("failed to update UDPIngress status because the object (%s/%s) changed. retrying...", namespace, name)
```

Comment:
```suggestion
				log.Debugf("failed to update UDPIngress (%s/%s) status, conflict err %v", namespace, name, err)
```

Reply:
:+1: changed in 2812019f36e6aed24e3021ec977f0475ab68e90d